roles/sro_deploy_art_bundle/templates/sro_sub.yml.j2

```diff
@@ -0,0 +1,12 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: special-resource-operator
+  namespace: openshift-special-resource-operator
+spec:
+  #channel: "{{ sro_channel }}"
+  channel: "4.9"
```

Comment:
not sure you want to keep `channel: "{{ sro_channel }}"` commented out

Reply:
That was for testing... it slipped.

Reply:
still there